crates/bevy_scene/Cargo.toml

```diff
@@ -8,6 +8,10 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
+[features]
+default = []
+serialize = ["dep:serde", "uuid/serde"]
```

Comment:
for consistency with other crates, what about `serde` instead of `serialize`? Since this feature actually does deserialize also.

Reply:
Okay I'll change the name of the feature, thank you!